tools/Customization/DevHome.Customization/Views/MainPageView.xaml

```diff
@@ -22,6 +22,7 @@
             AutomationProperties.AccessibilityView="Control"
             AutomationProperties.AutomationId="NavigateDevDriveInsightsCardButton"
             Command="{x:Bind ViewModel.NavigateToDevDriveInsightsPageCommand}"
+            Visibility="{x:Bind ViewModel.AnyDevDrivesPresent}"
```

Comment:
Probably a PM question, but do we want this to be setting Visibility or Enabled? If someone hasn't made a Dev Drive, maybe they'd want to see this is an option if they were to create one.

Reply:
This was discussed with the PMs. The purpose of the feature is to retain users on the dev drive and not bring new users into it. So it was decided not to display it if there are no drives.